ui/SpinInfoBar.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect, useState } from 'react';
+import { useDark } from './lib/useDark';
+
+export const SpinInfoBar = () => {
+  const [spinStack, setSpinStack] = useState<string[]>([]);
+  const dark = useDark();
+
+  useEffect(() => {
+    console.log('spin: setting up...');
+    let available = false;
+    const spin = new EventSource('/~_~/spin', { withCredentials: true });
```

Comment:
Worth noting that, with this and the `/~_~/slog` endpoint, and of course the eyre channel for herm subscriptions, this page now opens _three_ SSE connections to your ship's domain, which iirc browsers limit to some max amount per domain (not per tab/window).

Of course, multi-tab urbiters are already hurting because of this, but this will hurt them even more.

Wonder if it's necessary, or if there even is some way we could demux/unify these connections somehow.

Reply:
Maybe just make it toggleable and off by default?